pkg/feature/templates/servicemesh/authorino/base/authconfig.tmpl

```diff
@@ -0,0 +1,74 @@
+apiVersion: authorino.kuadrant.io/v1beta2
+kind: AuthConfig
+metadata:
+  name: odh-dashboard-protection
+  namespace: {{ .AppNamespace }}
+  labels:
+    {{ ReplaceChar .Auth.Authorino.Label "=" ": " }}
+spec:
+  hosts:
+  - {{ .AppNamespace }}.{{ .Domain }}
+  authentication:
+    cluster-users:
+      kubernetesTokenReview:
+        audiences:
+{{- range .Auth.Authorino.Audiences }}
+        - "{{ . }}"
+{{- end }}
+
+  authorization:
+
+    dashboard-access:
+      kubernetesSubjectAccessReview:
+        resourceAttributes:
+          group:
+            value: ""
+          name:
+            value: odh-dashboard
+          namespace:
+            value: {{ .AppNamespace }}
+          resource:
+            value: services
+          verb:
+            value: get
+        user:
+          selector: auth.identity.user.username
+      when:
+      - operator: neq
+        selector: context.request.http.path.@extract:{"sep":"/","pos":1}
+        value: notebook
+
+    notebook-access:
+      kubernetesSubjectAccessReview:
+        resourceAttributes:
+          group:
+            value: kubeflow.org
+          name:
+            value: ""
+          namespace:
+            selector: context.request.http.path.@extract:{"sep":"/","pos":2}
+          resource:
+            value: notebooks
+          verb:
+            value: get
+        user:
+          selector: auth.identity.user.username
+      when:
+      - operator: eq
+        selector: context.request.http.path.@extract:{"sep":"/","pos":1}
+        value: notebook
+
+  response:
+    success:
+      headers:
+        x-auth-data:
+          json:
```

Comment:
This is fine, but only so you know. Authorino also has [`plain`](https://docs.kuadrant.io/authorino/docs/features/#plain-text-responsesuccessheadersdynamicmetadataplain) now, in case you prefer just the username as the value in the header, rather than a stringified JSON.